packages/dom/src/tabbable.js

```diff
@@ -190,5 +188,5 @@ export function findNext( element ) {
 	// Remove all focusables before and including `element`.
 	const remaining = focusables.slice( index + 1 );
 
-	return first( filterTabbable( remaining ) );
+	return filterTabbable( remaining )[ 0 ];
```

Comment:
Should we handle the case where filterTabbable( remaining ) is undefined? (first handles it).

Reply:
I don't think so - `filterTabbable()` will always return an array, so it will never be `undefined`.